### ISSUE TITLE
How do I delete multiple dials? 

### ISSUE THREAD

**Issue by tockschmock** [fastaddons/GroupSpeedDial#171]
I managed to do a multiple selection (I recommend a Shift function to select dial X to dial Y / a dial range), but I cannot delete this selection.

**Comment by fastaddons** [comment]
Good point, I should add delete into the "Select dials" feature (it's in the main menu), which is currently used to move dials around.
Right now the fastest way to delete multiple dials is right click and press "c" key. That way you can quickly delete many dials.

**Comment by tockschmock** [comment]
Thanks
I am quite impressed by the app (coming from https://www.speeddial2.com/bookmarks).
Maybe my observations as a beginner with Group Speed Dial are helpful.

Most important to me is that local backups and sync between Chrome-OS and Windows (for others +Linux) works and that the dials are shownn nicely on mobile devices, because setting up dozens of dials with their thumbsnail is quite some effort..

A monthly fee for these basics would be an overkill for me (private use), while a one time payment (like speeddial2) is OK.
I will check the sync with Windows and my Android devices and hope I can switch to Group Speed Dial (seems more advanced and stable than Speeddial2).

Thank You

**Comment by fastaddons** [comment]
Thanks :),
I'll look into these issues probably in a few weeks. I have now my hands full with my new "Search Result Previews" addon which requires some urgent work right now.
